wiki/Ranking_Criteria/en.md

```diff
@@ -139,12 +139,12 @@ Glossary
 
 ### Rules
 #### Technical
-- **Metadata must be consistent across all difficulties of a beatmap set.**
-- **Guest mappers, storyboarders, skinners and hitsounders must be added to the tags of a beatmap set.** This is to give credit where credit is due and help others identify the main contributors of any given beatmap set. Usernames containing whitespaces should be written with underscores instead.
+- **Metadata must be consistent across all difficulties of a beatmapset.**
```

Comment:
you changed to "beatmapset" here but added "beatmap set" in BN rules, change all of em to beatmapset (might want to check other usages throughout the wiki too)

Reply:
adjusted